src/components/ButtonGroup/__tests__/buttonGroup-tests.jest.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import ButtonGroup from "../ButtonGroup";
+
```

Comment:
let's add here also a check for groupAriaLabel (set groupAriaLabel and check you success find it by using getByLabelText).